packages/ui/src/assets/css/button.css

```diff
@@ -41,7 +41,12 @@
   font-weight: 700;
   cursor: pointer;
   color: var(--_button-color);
-  transition: all 0.3s ease;
+  transition: all 0.2s ease;
```

Comment:
I think we have a transition duration variable defined somewhere. Could you please check that? We should be using that if there's any